### ISSUE TITLE
use external couchdb instance

### ISSUE THREAD

**Issue by icrc-fdeniger** [medic/helm-charts#35]
add an option to use an extern couchdb instance

**Comment by Paroksh1** [comment]
Hi @icrc-fdeniger, I’ve implemented support for using an external CouchDB instance via a `useExternal` flag in `values.yaml`, along with corresponding logic updates in the API, Sentinel, and Healthcheck deployments.

I've opened a PR for this.  
Requesting you to please assign this issue to me for tracking. Let me know if any changes or feedback are needed.

**Comment by dianabarsan** [comment]
I don't believe Medic will add support for external CouchDbs, as this opens opportunities for unsynced configs / startup code and can cause unforeseeable bugs and breaks (some discussion included in this forum post: https://forum.communityhealthtoolkit.org/t/extending-session-timeout-duration/4794/4). 

We will not include this in our default helm charts. 

I'm linking to this existent [PR](https://github.com/medic/helm-charts/pull/41) that adds this support for anyone who wants to implement this for their deployment.
